File: VB/App_Code/BootstrapChartDataItem.vb
''' <summary>
''' Summary description for BootstrapChartDataItem
''' </summary>
Public Class BootstrapChartDataItem
    Public Sub New(ByVal categoryName As String, ByVal categorySales As Decimal)
        Me.CategoryName = categoryName
        Me.CategorySales = categorySales
    End Sub

    Public Property CategoryName() As String

    Public Property CategorySales() As Decimal
End Class

File: CS/App_Code/BootstrapChartDataItem.cs
/// <summary>
/// Summary description for BootstrapChartDataItem
/// </summary>
public class BootstrapChartDataItem
{
    public BootstrapChartDataItem(string categoryName, decimal categorySales)
    {
        CategoryName = categoryName;
        CategorySales = categorySales;
    }

    public string CategoryName { get; set; }

    public decimal CategorySales { get; set; }
}